docs/chapter2-payload-image-format.rst

```diff
@@ -0,0 +1,659 @@
+Platform Init understands the concept of a compatible string and can make this
+determination.
+
+Where no compatible string matches, for any configuration, the FIT cannot be
+loaded and Platform Init should generate an error indicating that no
+configuration matches, along with information about what compatible string(s) it
+was looking for.
+
+Where no compatible strings are present, or Platform Init does not support this
+concept, the default configuration is used. In this case, there is no purpose to
+providing multiple images. Future revisions of this specification may use this
+to support configurations for different purposes, such as a recovery flow.
+
+
+2.3 FIT Schema
+--------------
+
+UPL conforms to the FIT schema but adds some additional requirements. These will
+be upstreamed to the FIT specification once finalized here.
+The structure is quite simple:
+
+* The root node holds an "images" subnode and a "configurations" subnode. These
+  subnodes have no properties.
+* The "images" node has one or more image subnodes
+* The "configurations" node has one or more configuration subnodes.
+  Configuration are collections of images
+
+::
+
+    / o image-tree
+        └── description = "image description"
+        └── timestamp = <12399321>
+        └── #address-cells = <1>
+        |
+        o images
+        | |
+        | o image-1 {...}
+        | o image-2 {...}
+        | ...
+        |
+        o configurations
+          └── default = "conf-1"
+          |
+          o conf-1 {...}
+          o conf-2 {...}
+          ...
+
+Below are the allowable properties for each node. Required properties, which
+must appear a valid UPL, are marked "(R)". Optional properties are marked "(O)".
+Platform Init must support both classes of properties.
+
+Nodes with properties are shown in the tables below:
+
+.. note:: 
+
+    Usage legend: R=Required, O=Optional, OR=Optional but Recommended,
+    SD=See Definition
+
+**Node: / (Root Node)**
+
+.. tabularcolumns:: | p{4cm} p{0.75cm} p{4cm} p{6.5cm} |
+.. table:: Node: /
+
+   ======================= ===== ===================== ===============================================
+   Property Name           Usage Value Type            Definition
+   ======================= ===== ===================== ===============================================
+   ``description``         R     ``<string>``          General description of the Loader. This
+                                                       may be displayed to the user.
+   ``timestamp``           R     ``<u32>``             Last image modification time, as seconds
+                                                       in POSIX time format\ :sup:`1`. This is updated by
+                                                       any tool which creates or changes the FIT.
+   ``size``                OR    ``<u32>``             Specifies current serial device speed in bits
+                                                       per second
```

Comment:
It looks the description doesn't match with name "size". please double check.

Reply:
Nice catch! Done.